client/dashboard/emails/layout.tsx

```diff
@@ -20,7 +20,7 @@ export const Layout = ( { children }: PropsWithChildren ) => {
 						<>
 							<Button
 								className="emails__add-email-forwarder"
-								variant="link"
+								variant="secondary"
```

Comment:
It's intented... See https://github.com/Automattic/wp-calypso/pull/106069#discussion_r2382267437 🫠